src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java

```diff
@@ -73,11 +84,21 @@ protected Object doWork() {
         return indexFile.getAbsolutePath();
     }
 
+    private void writeTabixIndex(final TabixIndex idx, final File idxFile) throws IOException {
+        if (!idxFile.getAbsolutePath().endsWith(TabixUtils.STANDARD_INDEX_EXTENSION)) {
+            //Creating tabix indices with a non standard extensions can cause problems so we disable it
+            throw new UserException("File " + featureFile + " requires a tabix index file with a \"" + TabixUtils.STANDARD_INDEX_EXTENSION + "\" extension");
+        }
+        try (final LittleEndianOutputStream stream = new LittleEndianOutputStream(new BufferedOutputStream(new BlockCompressedOutputStream(idxFile)))){
```

Comment:
The tbi index file *itself* is block-compressed? Are you sure about this @akiezun?